apps/homepage/__tests__/e2e/test.cy.tsx

```diff
@@ -1,16 +1,38 @@
-describe('First test', () => {
-  it('Should check Header see through out pages', () => {
-    cy.visit('/');
+describe('Localization Route Tests', () => {
```

Comment:
This test should be in a test file that is specific for `localization`, it could be something like:
`localization.cy.tsx`